src/repositories/ColorRepository.ts

```diff
@@ -80,7 +79,7 @@ export function mapColorApiToFrontifyColor(color: ColorApi): FrontifyColor {
     };
 }
 
-export function mapFrontifyColorCreateToColorApiCreate(color: FrontifyColorCreate): ColorApiCreate {
+export function mapFrontifyColorCreateToColorApiCreate(color: ColorCreate): ColorApiCreate {
```

Comment:
Should update method name as well:
```suggestion
export function mapColorCreateToColorApiCreate(color: ColorCreate): ColorApiCreate {
```